cmd/launcher/desktop.go

```diff
@@ -180,3 +193,8 @@ func defaultSocketPath() string {
 
 	return filepath.Join(os.TempDir(), fmt.Sprintf("%s_%d", socketBaseName, os.Getpid()))
 }
+
+// TODO pick a more sensible default
+func defaultIconPath() string {
+	return os.TempDir()
+}
```

Comment:
Maybe I leave this for now (it works fine) and update to pull from the menu path once https://github.com/kolide/launcher/pull/984/files is merged?